corehq/apps/formplayer_api/smsforms/api.py

```diff
@@ -276,16 +276,18 @@ def user_id(self):
 
         return self._user_id
 
-    def get_raw_instance(self):
+    def get_raw_instance(self, respect_relevancy=False):
```

Comment:
This doesn't seem quite right to me. This method submits the POST to HQ, so I'd consider it to be an exception rather than the common path to be including nodes which aren't passing their relevancy condition.

Reply:
Yeah I was unsure on what the default case should be. I went with not respecting relevancy since that is the current default behavior on formplayer's end. Though looking again, this method is only used in the context you mentioned, so defaulting to True does make sense. Made change on aa19f174e3212da5a80fa9ba234b598e3dc33334